test/services/websites/create_test.rb

```diff
@@ -6,32 +6,47 @@ class CreateTest < ActiveSupport::TestCase
       user = users(:hari)
       collection = user.collections.create!(name: 'My Collection')
 
+      params = { website: { url: 'http://twitter.com', collection_id: collection.id } }
+
+      Websites::Create.call(params, user)
+
+      user.reload
+      assert_equal 1, user.site_number, 'site number'
```

Comment:
Does this field contain number of websites a user has? Then can we call it `websites_count` instead? This seems an awkward name for the field.

(Do this in a separate PR rather than mixing these.)